test/concerns/scheduled_test.rb

```diff
@@ -0,0 +1,18 @@
+require "test_helper"
+
+class ScheduledTest < ActiveSupport::TestCase
+  test "Ants:Scheduled" do
+    Book.create(title: 'The Art of War', published_at: Time.zone.now)
+    Book.create(title: 'Alphabetic writing', published_at: Time.zone.now + 3)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.